dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +281,22 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name="static-check")
+@click.option('-t', '--type', type=click.Choice(PRE_COMMIT_LIST), multiple=True)
+def static_check(verbose: bool, type: Tuple[str]):
+    if check_package_installed('pre_commit'):
+        command_to_execute = ['pre-commit', 'run']
+        for single_check in type:
+            command_to_execute.append(single_check)
+        command_to_execute.append(EXTRA_STATIC_CHECK_OPTIONS)
+        output = run_command(
+            command_to_execute, suppress_raise_exception=True, capture_output=True, text=True
```

Comment:
Yeah. Capture-output is not needed here - I think it's better if the pre-commit prints directly to stdout.